lib/event_id_populator.rb

```diff
@@ -0,0 +1,7 @@
+class EventIdPopulator
+  def self.populate
+    EventLog::EVENTS.each do |event|
+      EventLog.where(event: event.description, event_id: nil).update_all(event_id: event.id)
```

Comment:
Will this cover all the events? Have some of the events changed their description?